client/app/scripts/reducers/root.js

```diff
@@ -421,7 +421,12 @@ export function rootReducer(state = initialState, action) {
       // highlight edge
       state = state.update('highlightedEdgeIds', (highlightedEdgeIds) => {
         highlightedEdgeIds = highlightedEdgeIds.clear();
-        return highlightedEdgeIds.add(action.edgeId);
+        highlightedEdgeIds = highlightedEdgeIds.add(action.edgeId);
+        if (action.bidirectional) {
+          const opposite = action.edgeId.split(EDGE_ID_SEPARATOR).reverse().join(EDGE_ID_SEPARATOR);
+          highlightedEdgeIds = highlightedEdgeIds.add(opposite);
+        }
+        return highlightedEdgeIds;
```

Comment:
Hmm okay so my original thought would be not to add a bunch of extra edges that are not bidirectional to the `highlightedEdgeIds` list. It would be confusing to someone looking at the state tree and seeing a bunch of edges that don't exist.

But I guess the precedent has already been set and that would make this simpler.